### PR TITLE
updated config.yml based on plots2 model

### PR DESCRIPTION
**Pull request by Uzay-G**
#357 
I updated the config file based on this [example](https://github.com/publiclab/plots2/blob/master/.github/config.yml) as part of my first Google CodeIn task. I changed some links that were specific to the plots2 repository. I am not sure if I should have included the ```<details>``` section.

I hope I didn't make any mistakes! 😄

### CODE REVIEW

**Comment by ananyaarun** [comment]
Hi @Uzay-G , Great job on your first task :smiley: 
Next time just link the issue you are solving in the PR template  like I edited :)

**Comment by keshav234156** [comment]
@Uzay-G also please have a look at https://github.com/publiclab/plots2/issues/6808

**Comment by ananyaarun** [comment]
Hey @Uzay-G , This PR is just fine, but were you able to locally install the repo ?
We recommend you to locally install the repo first (do the installation tasks ) and then do a beginner task .
Let me know id you need any more help !
Thanks

**Comment by Uzay-G** [comment]
Hey @ananyaarun, thank you for your help! Next time I will link the issue. I've managed to install the plots2 repository and am going to clone this one too. Thanks again.

**Comment by ananyaarun** [comment]
@Uzay-G great !!

**Comment by keshav234156** [comment]
@jywarren @rexagod I think It's is good to merge.

File: .github/config.yml [hunk 1]
```diff
@@ -4,22 +4,36 @@
...
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄    
+ It would be great if you can tell us your Twitter handle so we can thank you properly?
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: |
  Congrats on merging your first pull request! 🙌🎉⚡️ 
- Your code will likely be published to https://publiclab.org in the next few days. 
- In the meantime, can you tell us your Twitter handle so we can thank you properly?
+ Your code will likely be published to PublicLab.org in the next few days, but first it will be published to https://stable.publiclab.org/ (it will take some minutes for this to load, and until then you may see logs from the build process). Please test out your work on this testing server and report back with a comment that all has gone well! In the meantime, can you tell us your Twitter handle so we can thank you properly?
```
Comment (jywarren) [inline .github/config.yml]: Ah this won't happen quite as fast as on `plots2` -- it'll trigger Dependabot to open a new version in a PR on `plots2` /if/ we publish it to NPM, and then it'll need to be merged there. So, maybe just end with "in a few days"?

Like here: 

https://github.com/publiclab/leaflet-blurred-location/blob/1bff37cb96518bd87192277b14c290cba3c138ba/.github/config.yml#L20

```suggestion
 Your code will likely be published to PublicLab.org in the next few days. In the meantime, can you tell us your Twitter handle so we can thank you properly?
```

File: .github/config.yml [hunk 2]
```diff
@@ -4,22 +4,36 @@
...
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
- Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
- There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+ Thanks for opening this pull request!  This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+ `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
+ Also please refer [here](https://github.com/publiclab/PublicLab.Editor#installation) for installation help 💿    
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄    
+ It would be great if you can tell us your Twitter handle so we can thank you properly?
```
Comment (jywarren) [inline .github/config.yml]: We may also want to highlight that we'd like the version number to be bumped in `package.json` (pointing to the file URL) and that we'd like the changes built into the `/dist/` folder as well?

This is awesome!

Comment (Uzay-G) [reply .github/config.yml]: Alright I took care of it [here](https://github.com/publiclab/PublicLab.Editor/commit/4922436531646df0c6ab038f7809c14c7e1fcdcd). Thanks!

**Comment by jywarren** [comment]
Amazing!!! Thanks so much!!

**Pull request merged**